### PR TITLE
[3.0.2-prepare] Update version number from 3.0.1 to 3.0.2

### PR DESCRIPTION
**Pull request by EricGao888**
## Purpose of the pull request

* Update version number from 3.0.1 to 3.0.2

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: deploy/kubernetes/dolphinscheduler/Chart.yaml [hunk 1]
```diff
@@ -35,11 +35,11 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.0.0
+version: 2.0.1
```
Comment (zhongjiajie) [inline deploy/kubernetes/dolphinscheduler/Chart.yaml:38]: Is it a unrelated change?

File: docs/configs/site.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export default {
   port: 8080,
   domain: 'dolphinscheduler.apache.org',
   copyToDist: ['asset', 'img', 'file', '.asf.yaml', 'sitemap.xml', '.nojekyll', '.htaccess', 'googled0df7b96f277a143.html'],
-  docsLatest: '3.0.1',
```
Comment (zhongjiajie) [inline docs/configs/site.js]: This file should not change, we use change branch `dev` one instead of branch `3.0.2-prepare` one

File: docs/configs/index.md.jsx [hunk 3]
```diff
@@ -64,6 +64,7 @@ const docsSource = {
   '2.0.5': docs205Config,
   '3.0.0': docs300Config,
   '3.0.1': docs301Config,
+  '3.0.2': docs302Config,
```
Comment (zhongjiajie) [inline docs/configs/index.md.jsx]: So as this file, we should change branch `dev` before we announce instead of this step

**Review by zhongjiajie — CHANGES_REQUESTED** [review]

File: docs/configs/index.md.jsx (hunk 3 above)
Comment (zhongjiajie) [reply docs/configs/index.md.jsx]: you can see more detail in section **Update Document**

File: deploy/kubernetes/dolphinscheduler/Chart.yaml (hunk 1 above)
Comment (EricGao888) [reply deploy/kubernetes/dolphinscheduler/Chart.yaml:38]: The comment says we should increment `version` if we change `app version`. Did I get it wrong?
image

File: docs/configs/site.js (hunk 2 above)
Comment (EricGao888) [reply docs/configs/site.js]: Got it, thx : )

File: docs/configs/index.md.jsx (hunk 3 above)
Comment (EricGao888) [reply docs/configs/index.md.jsx]: > you can see more detail in section _Update Document_

Looks like we need some updates for the docs. I will submit a PR later to add more instructions. 
image

**Comment by EricGao888** [comment]
@zhongjiajie The link has broken since we move pyds into another repo, `https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-python/pydolphinscheduler/RELEASE.md#to-pypi`

**Comment by zhongjiajie** [comment]
@EricGao888  I fix the dead link of this PR
